src/mlem.rs

```diff
@@ -263,15 +287,20 @@ where
                 tof_peak, &tof
             );
 
+            // LOR struct has additive_correction but ignore for now as LOR unmutable.
```

Comment:
> The additive_correction in LOR was added to try out an approach that uses a pre-processing step. I think we agreed not to go down that path, at least for now.
Yes, I was just making notes for the future here. It might be awkward to implement but seems elegant to be able to get these values at LOR load time. Not relevant at the moment though.

I'll take a look at the literature.